pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md

```diff
@@ -49,8 +49,11 @@ Follow these steps to install the DKP catalog from the CLI.
 
 ## Workspace DKP catalog applications
 
-| NAME                          | APP ID                |
-| ----------------------------- | --------------------- |
-| kafka-operator-0.20.0         | kafka-operator        |
-| spark-operator-1.1.6          | spark-operator        |
-| zookeeper-operator-0.2.13     | zookeeper-operator    |
+| Name                      | App ID             | Supported<br>Kubernetes<br>version |
+|---------------------------|--------------------|------------------------------------|
+| kafka-operator-0.20.0     | kafka-operator     | 1.21                               |
+| kafka operator-0.20.2     | kafka-operator     | 1.21<br>1.22                       |
```

Comment:
Why do we use a range in the 2.3 (`1.21 - 1.23`) table but use separate versions in 2.2 table (`1.21<br>1.22`)? Can we make these table formats consistent?

Reply:
I forgot there were patch releases between 1.21 and 1.22 hehehe. Thanks for pointing that out!